bot_instructions.rb

```diff
@@ -1,11 +1,15 @@
-class BotInstructions
-  @@instructions = YAML.load_file('instructions.yml')
+module BotInstructions
+  extend self
```

Comment:
In this moment I prefer module. 

You comments where very useful, thanks a lot!